core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -752,25 +793,35 @@ class SparkSubmitSuite
 
   test("downloadFile - invalid url") {
     intercept[IOException] {
-      SparkSubmit.downloadFile("abc:/my/file", new Configuration())
+      val sparkConf = new SparkConf()
```

Comment:
Should we create a `testDownloadFile()` function and merge the duplicated test code?